docs/manual.rst

```diff
@@ -1098,9 +1098,37 @@ differently.
   >>> b == c
   False
 
-.. method:: object.equals_exact(other, tolerance)
-
-  Returns ``True`` if the object is within a specified `tolerance`.
+.. method:: object.equals_exact(other, tolerance=0.0, normalize=False)
+
+    Returns True if the geometries are structurally equivalent within a given
+    tolerance.
+
+    This method uses exact coordinate equality, which requires coordinates
+    to be equal (within specified tolerance) and in the same order for
+    all components (vertices, rings, or parts) of a geometry. This is in
+    contrast with the :meth:`~object.equals` function which uses spatial
+    (topological) equality and does not require all components to be in the
+    same order.
+    Because of this, it is thus possible for :meth:`~object.equals` to be True
```

Comment:
```suggestion
    Because of this, it is possible for :meth:`~object.equals` to be ``True``
```

Sorry I didn't catch this sooner, but inline code like True needs to have double-backticks in RST.  This isn't consistent throughout Shapely yet, but it looks like these should be double-backticks in the docstrings as well since they get pulled into the docs (or will in the future, for this one).

Also, this can be part of the prior paragraph ("This method uses...") instead of a new paragraph.